docs/05_Conversation_History.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.    history = conversation.get_history(max_tokens=500)
</pre><p>Add working code — don't leave API errors inline, this looks awful when someone reads it for reference!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/agentai/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I hear you.